tubemq-server/src/main/java/com/tencent/tubemq/server/common/utils/Bytes.java

```diff
@@ -84,25 +85,21 @@ public static String toStringBinary(final byte[] b) {
 
     public static String toStringBinary(final byte[] b, int off, int len) {
         StringBuilder result = new StringBuilder();
-        try {
-            String first = new String(b, off, len, "ISO-8859-1");
-            for (int i = 0; i < first.length(); ++i) {
-                int ch = first.charAt(i) & 0xFF;
-                if ((ch >= '0' && ch <= '9') || (ch >= 'A' && ch <= 'Z') || (ch >= 'a' && ch <= 'z')
-                        || " `~!@#$%^&*()-_=+[]{}\\|;:'\",.<>/?".indexOf(ch) >= 0) {
-                    result.append(first.charAt(i));
-                } else {
-                    result.append(String.format("\\x%02X", ch));
-                }
+        String first = new String(b, off, len, StandardCharsets.ISO_8859_1);
```

Comment:
@slievrly Why not keep exception catches? I see an UnsupportedEncodingException may be thrown when constructing a String object.

- - - - - - - 
为什么不保留异常捕获呢？ 我看到构造String对象时可能会抛出UnsupportedEncodingException。

Reply:
@gosonzhang before modification, the charset type use String, it may be  unknown charset type, so it can throw UnsupportedEncodingException. use StandardCharsets  enum type always known type，so it will not throw UnsupportedEncodingException.
![image](https://user-images.githubusercontent.com/8758457/69474117-bb60fc00-0df7-11ea-8eae-96d78d1f2976.png)
In addition, take a step back, as a util class, the exception is swallowed, obviously not suitable, should be handed over to the upper-level business to deal with.
---
修改之前传入的字符类型参数是字符串，所以它可能传入的是一个未知不合法的字符格式，这里会抛出不支持的异常，修改后使用枚举来标识字符类型，那么一定是已知类型，不会抛出不支持异常，方法的定义如图。
另外，退一步讲，作为一个util类，觉得这里直接将这种不支持字符异常吞掉不太合适，util类没有能力处理这种类型的异常，应该抛给上层业务去处理。util作为通用的处理逻辑，吞掉异常对所有引用场景是否都合适？

Reply:
You're right, thanks!